src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -265,6 +265,7 @@ const SendAssetsForm = ({ txConfig, onComplete, protectTxButton, isDemoMode }: P
   const [isEstimatingNonce, setIsEstimatingNonce] = useState(false); // Used to indicate that interface is currently estimating gas.
   const [isResolvingName, setIsResolvingDomain] = useState(false); // Used to indicate recipient-address is ENS name that is currently attempting to be resolved.
   const [fetchedNonce, setFetchedNonce] = useState(0);
+  const [isSendMax, toggleIsSendMax] = useState(false);
```

Comment:
Can we disable the amount input while this is active? It seems weird to be able to change while I am supposed to be sending max

Reply:
i did so